test/spec/videoCache_spec.js

```diff
@@ -1,8 +1,7 @@
-import 'mocha/mocha';
```

Comment:
Guessing this is an unused import

Reply:
I sure hope so, I can't imagine what it would be doing where it'd need to be included only for its side-effects... either way I removed it and the tests still passed so... good?

Reply:
👍